scripts/run_validation.py

```diff
@@ -173,13 +175,32 @@ def run_validation(args: namedtuple):
     datasets = get_datasets(data_service, data_path)
 
     start = time.time()
-    pool = Pool(args.pool_size)
-    results = pool.map(
-        partial(validate_single_rule, shared_cache, data_path, datasets, args),
-        rules,
-    )
-    pool.close()
-    pool.join()
+    results = []
+
+    # run each rule in a separate thread
```

Comment:
Not thread but a process, it is important to understand the difference

Reply:
fixed